src/main/java/uk/gov/mca/beacons/service/model/Registration.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.mca.beacons.service.model;
+
+import java.util.UUID;
+import javax.persistence.*;
+
+@Entity
+public class Registration {
+
+  @Id
+  private UUID id;
+
+  public UUID getId() {
+    return UUID.randomUUID();
+  }
+}
```

Comment:
Currently very sparse pending database change (see comment about migration in PR).

UUID would be replaced by `@GeneratedValue(strategy = GenerationType.AUTO)` when class is properly mapped to database. 

Many missing fields here (owner details, beacon details, uses, emergency contacts etc.).  All to be included once data model is updated.